src/components/utils/error/withErrorAnnouncer.js

```diff
@@ -8,73 +8,67 @@
  */
 import React, { useCallback, useState } from "react";
 
-import {
-    announce,
-    AnnouncerConstants,
-    formatMessage,
-    withI18N,
-} from "@cyverse-de/ui-lib";
+import { useTranslation } from "react-i18next";
+
+import { announce, AnnouncerConstants } from "@cyverse-de/ui-lib";
 import { Button, Typography, useTheme } from "@material-ui/core";
-import { injectIntl } from "react-intl";
 
 import DEErrorDialog from "./DEErrorDialog";
 import ids from "../ids";
-import messages from "../messages";
 
-const withErrorAnnouncer = (WrappedComponent) =>
-    withI18N(
-        injectIntl((props) => {
-            const theme = useTheme();
-            const { intl } = props;
+const withErrorAnnouncer = (WrappedComponent) => {
+    const WithErrorAnnouncerComponent = (props) => {
+        const theme = useTheme();
+        const { t } = useTranslation("util");
 
-            const [errorDialogOpen, setErrorDialogOpen] = useState(false);
-            const [errorObject, setErrorObject] = useState(null);
+        const [errorDialogOpen, setErrorDialogOpen] = useState(false);
+        const [errorObject, setErrorObject] = useState(null);
 
-            const viewErrorDetails = useCallback(
-                () => (
-                    <Button
-                        variant="outlined"
-                        onClick={() => setErrorDialogOpen(true)}
+        const viewErrorDetails = useCallback(
+            () => (
+                <Button
+                    variant="outlined"
+                    onClick={() => setErrorDialogOpen(true)}
+                >
+                    <Typography
+                        variant="button"
+                        style={{ color: theme.palette.error.contrastText }}
                     >
-                        <Typography
-                            variant="button"
-                            style={{ color: theme.palette.error.contrastText }}
-                        >
-                            {formatMessage(intl, "details")}
-                        </Typography>
-                    </Button>
-                ),
-                [intl, theme.palette.error.contrastText]
-            );
+                        {t("details")}
+                    </Typography>
+                </Button>
+            ),
+            [t, theme.palette.error.contrastText]
```

Comment:
@ash3rz - This change worked. Is this what you were referring to ?

Reply:
Yup, this is the exact case - so if you can get Preferences to show this error announcer and everything looks ok, then we're good 👍